transcribe/transcribe.py

```diff
@@ -8,17 +8,31 @@
 from whisper.utils import get_writer
 
 class Transcriber:
-    def __init__(self, directory, output, language_model, language, fp16):
+    def __init__(self, directory, output, language_model, language, fp16, ignore_existing=False):
         self.directory = directory
         self.output = output
         self.model = language_model
         self.language = language
         self.fp16 = fp16
+        self.ignore_existing = ignore_existing
+        if self.ignore_existing:
+            self.existing = self.__cache_existing()
+        else:
+            self.existing = []
+
+    def __cache_existing(self):
+        existing = []
+        for dirpath, dirnames, filenames in os.walk(self.output):
+            for filename in filenames:
+                if filename.endswith('.vtt'):
+                    existing.append(f'{os.path.splitext(filename)[0]}_{self.model}')
```

Comment:
I think this should be:

```suggestion
                    existing.append(f'{os.path.splitext(filename)[0]}')
```